toml-to-ical-bin/src/main.rs

```diff
@@ -35,24 +35,28 @@ struct Opt {
     output: Option<PathBuf>,
 }
 
-struct Session<'p> {
-    base_path: &'p Path,
+struct Session {
+    base_path: PathBuf,
 }
 
-impl<'p> Session<'p> {
-    fn new(base_path: &'p Path) -> Self {
+impl Session {
+    fn new(base_path: PathBuf) -> Self {
         Self { base_path }
     }
 }
 
-impl<'p> External for Session<'p> {
+impl External for Session {
     type Error = Error;
 
     fn load_without_includes(&self, path: &Path) -> Result<Calendar, Error> {
```

Comment:
It might be better to just thread `base_path` through as an argument.

Reply:
I believe I did what you were asking